TLM/TLM/Lifecycle/SerializableDataExtension.cs

```diff
@@ -79,16 +83,21 @@ public class SerializableDataExtension
                 if (TMPELifecycle.InGameOrEditor()) {
                     // Always force default options on new game
                     // See: https://github.com/CitiesSkylinesMods/TMPE/pull/1425
-                    byte[] options = TMPELifecycle.IsNewGame
-                        ? null
-                        : SerializableData.LoadData("TMPE_Options");
-
-                    if (!OptionsManager.Instance.LoadData(options ?? new byte[0])) {
-                        loadingSucceeded = false;
+                    if (!TMPELifecycle.IsNewGame) {
+                        if (Version <= 3) {
```

Comment:
Looks like a bug, or you should swap bodies because `LoadData(data)` is loading XML Options (`OPTIONS_ID`) 
which does not exist if version is **equal or lower than 3**